app/assets/stylesheets/events.scss

```diff
@@ -1,3 +1,72 @@
 // Place all the styles related to the Events controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: https://sass-lang.com/
+.event {
```

Comment:
I forgot before, 
I think it's better to use plural here as well as we will be showing multiple events under this class.